infra/docker-evm-blockhain/Dockerfile

```diff
@@ -3,18 +3,47 @@ FROM ethereum/client-go:alltools-latest
 # Install required packages
 RUN apk add --no-cache curl jq
 
+# ADD . /app
 # Set the working directory
-WORKDIR /root
+# WORKDIR /app
 
 # Copy the password file
-COPY root-password.txt /root/root-password.txt
+COPY root-password.txt /app/root-password.txt
 
 # Copy the startup script
-COPY start.sh /root/start.sh
+COPY start.sh /app/start.sh
+
+# Set the proper permissions for the script
+RUN chmod -R 777 /app
+RUN chmod -R 777 /bin/sh
+RUN chmod -R 777 /bin/busybox
```

Comment:
Do we need 777 permission here? AFAIR we added 777 permission during the debug session, but do we really need 777 permission here?